.github/workflows/audio-integration.yml

```diff
@@ -4,7 +4,7 @@
     pull_request:
       branches:
         - main
-        - release-1.x
+        - 'release-**.x'
```

Comment:
This would test against any of the older major releases for e.g. if that is 1.x, 2.x or 3.x. How would I exclude a previous major release in case of a very old version that is deprecated?

Reply:
We can delete the branch basically. If we deprecate a version there is no reason to keep the branch.

Reply:
There's also syntax to exclude branches if we want to, if we put in !release-1.x that will exclude that branch from being ran on.